### ISSUE TITLE
Not able to get price data?

### ISSUE THREAD

**Issue by dipespandey** [miraclesu/uniswap-sdk-go#16]
Don't I have to add my infura(or others) provider URL to get the blockchain data for computing price?
Can you please add an example to extract the price of WETH in USDC?

**Comment by jon4hz** [comment]
No, this is an offchain library. If you want to use it with live data, you'll have to fetch it first using ethclient e.g.
